src/timer/Timer.js

```diff
@@ -1,17 +1,18 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function() {
-  var instance = this;
-  var timerInterval = setInterval(function() {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    const timerInterval = setInterval((instance) => {
+      if (instance.seconds === 0) {
```

Comment:
Remember the reason we use an arrow function here is so 'this' remains bound to the timer object and not the new function start. That way we can access this.seconds on line 8. Since we remove the variable instance  instance.seconds is undefined.